HIPs/firefly.md

```diff
@@ -0,0 +1,379 @@
+---
+layout: default
+title: HIP Template
+parent: Hyperledger Firefly
```

Comment:
These two lines should be changed to ensure that the https://hyperledger.github.io/hyperledger-hip/ shows the proposal in the correct location:
```
title: Hyperledger Firefly
parent: Hyperledger Improvement Proposals
```

Reply:
thanks @tkuhrt for suggesting that. updated as suggested.